js/admin.js

```diff
@@ -0,0 +1,31 @@
+jQuery( document ).ready(function($) {
+	$('#check-status').click(function(){
+		$(".theme-check-report").empty();
+		themeCheckRunPHPCS();
+	});
+	function themeCheckRunPHPCS(){
+		$.ajax({
+			url: themeCheck.ajax_url,
```

Comment:
I didn't see the comment you've made before, I was commenting on my first comment about not working. The whole plugin stopped working, not just this code. I'll try to install it cleanly fresh and then test the changes.